hotel/views.py

```diff
@@ -1 +1,25 @@
-from django.shortcuts import render
+import django
+from django.shortcuts import render, redirect
+from django.contrib.auth import login
+from django.contrib import messages
+from django.http.response import HttpResponse, HttpResponseRedirect
+from django.urls import reverse
+from .forms import NewUserForm
+
+# Create your views here.
+def register_request(request):
+    if request.method == "POST":
+        form = NewUserForm(request.POST)
+        if form.is_valid():
+            user = form.save()
```

Comment:
ở đây nên dùng thêm commit nếu ở bên dưới có set và save thêm lần nữa nha e
```suggestion
            user = form.save(commit=False)
```